integration-tests/tests/network/routing.rs

```diff
@@ -103,6 +103,8 @@ fn ping_jump() {
     runner.push(Action::AddEdge(0, 1));
     runner.push(Action::AddEdge(1, 2));
     runner.push(Action::CheckRoutingTable(0, vec![(1, vec![1]), (2, vec![1])]));
+    runner.push(Action::CheckRoutingTable(1, vec![(0, vec![0]), (2, vec![2])]));
+    runner.push(Action::CheckRoutingTable(2, vec![(0, vec![1]), (1, vec![1])]));
```

Comment:
What do these checks do?

Reply:
@bowenwang1996 In this test we start 3 nodes. We were checking the routing table only one one 0. We should check it on all nodes 0,1,2.

Reply:
@bowenwang1996 If anything happens, we will know whenever there is an issue with routing table or with ping pong.